src/state.rs

```diff
@@ -146,6 +147,31 @@ impl PreKeyRecord {
     }
 }
 
+/// Helper function for generating N prekeys.
+/// Returns a list of PreKeyRecords.
+///
+/// # Example
+///
+/// ```
+/// from signal_protocol import curve, state
+/// prekeypair = curve.KeyPair.generate()
+/// prekeyid = 1
+/// manykeys = state.generate_n_prekeys(100, prekeyid, prekeypair)  # generates 100 keys
+/// ```
+#[pyfunction]
+pub fn generate_n_prekeys(n: u16, id: PreKeyId) -> Vec<PreKeyRecord> {
+    let mut keyvec: Vec<PreKeyRecord> = Vec::new();
+    let mut i: u32 = id;
+    for _n in 0..n {
+        let keypair = KeyPair::generate();
```

Comment:
When I play with this on the Python end, it still looks suspiciously like we're not getting a new KeyPair:

```python
In [2]: x = 1

In [3]: while x < 50:
   ...:     print(manykeys[x].id())
   ...:     print(manykeys[x].key_pair())
   ...:     x += 1
   ...: 
2
<KeyPair object at 0x733690406c38>
3
<KeyPair object at 0x733690406c38>
4
<KeyPair object at 0x733690406c38>
5
<KeyPair object at 0x733690406c38>
6
<KeyPair object at 0x733690406c38>
7
```

Reply:
*Howeeever*, the test I wrote just passed.

```python
prekeyrecords = state.generate_n_prekeys(256, pre_key_id)
    assert len(prekeyrecords) == 256
    assert prekeyrecords[12].id() != prekeyrecords[13].id()
    assert prekeyrecords[14].key_pair() != prekeyrecords[15].key_pair()
```